hail/python/test/hail/vds/test_vds.py

```diff
@@ -826,6 +827,33 @@ def test_split_sparse_roundtrip():
     assert vds2.reference_data._same(vds_split.reference_data.drop('ref_allele'))
 
 
+def test_split_alters_ref_gt():
+    vds_base = hl.vds.read_vds(os.path.join(resource('vds'), '1kg_chr22_5_samples.vds'))
+
+    # no lgt or gt
+    vds_split = hl.vds.split_multi(vds_base)
+    assert 'GT' not in vds_split.reference_data.entry, vds_split.reference_data.entry
```

Comment:
Was the intention in all these assertions to look in both `vds_split.reference_data.entry` and `vds_split.variant_data.entry`?

Reply:
no, just the reference data

Reply:
Ahh, the second `vds_split.reference_data.entry` is the "print" of the assert. The paren-less syntax was throwing me!